lib/docs/filters/padrino/entries.rb

```diff
@@ -0,0 +1,17 @@
+module Docs
+  class Padrino
+    class EntriesFilter < Docs::EntriesFilter
+      def get_name
+        node = at_css('h1')
+        result = node.content.strip
+        result << ' event' if type == 'Events'
+        result << '()' if node['class'].try(:include?, 'function')
+        result
+      end
+
+      def get_type
+        return 'Ruby'
```

Comment:
Ok. We need to remove these `Class: ` in the entries' names. Also we should add entries (using `additional_entries`) for all the methods. Right now it looks like it's only indexing modules/classes. See [here](https://github.com/Thibaut/devdocs/blob/ebd6a2204da031328c1ba707e9d119e3969b4146/lib/docs/filters/rdoc/entries.rb#L30-L48) for an example. Documentation is [here](https://github.com/Thibaut/devdocs/wiki/Filter-Reference#entriesfilter).

> The navigation in the HTML is still missing (think some JS is missing there)

You're not supposed to open the HTML files directly. The app doesn't work like that. Run `rackup` in the root directory. That will boot the server. Then you can access the app at http://localhost:9292/